packages/sdk/src/oracle/store/has.ts

```diff
@@ -0,0 +1,22 @@
+import Read from "./read";
+// we only ignore this specific error type explicitly emitted from reader class, meaning value is undefined
+import { ignoreExistenceError as ignoreError } from "../errors";
```

Comment:
nit: can we keep the ignoreExistenceError just to make it clear that we're _only_ ignoring a specific type of error? If I were reading the code and saw ignoreError, I would probably misinterpret what was being done.

Reply:
yea can do that